pkg/manager/manager.go

```diff
@@ -107,6 +108,8 @@ func Login(ctx context.Context, password, address string) (user string, pass str
 			} else {
 				logrus.Debugf("tokenRequest.Status.Token is empty")
 			}
+		} else if passwordIsSpecified && strings.Contains(err.Error(), "404") {
```

Comment:
nit: we could also return a custom `notFound` error from `httpGet` and use `errors.Is(...)` to be a bit more idiomatic.